Gopkg.lock

```diff
@@ -1165,23 +1165,23 @@
   version = "v1.0.5"
 
 [[projects]]
-  digest = "1:17942a7b637d6adc4010c82e1a75a52ef31e7487520aeef80faa33bda08c192c"
+  digest = "1:f3cd75f81ee2b143bd354d906370261e94fc58e66dd5d632e1d6cde828338741"
   name = "github.com/spf13/cobra"
   packages = [
     ".",
     "doc",
   ]
   pruneopts = "NUT"
-  revision = "a1f051bc3eba734da4772d60e2d677f47cf93ef4"
-  version = "v0.0.2"
+  revision = "ef82de70bb3f60c65fb8eebacbb2d122ef517385"
+  version = "v0.0.3"
```

Comment:
Any reason not to grab 0.0.4?

Reply:
Should have tagged that in the .toml, but happened to be looking at the .lock.

Reply:
To minimize risk. This is the version used in master currently, so it has some testing.